src/test/java/api/item/ItemLastCheckInTests.java

```diff
@@ -152,8 +174,31 @@ public void shouldBeAbleCheckinItemWithoutLoanMultipleTimes()
     lastCheckIn = itemsClient.get(item.getId()).getJson()
       .getJsonObject("lastCheckIn");
 
-    assertThat(lastCheckIn.getString("dateTime"), is(secondCheckInDate.toString()));
+    actualCheckinDateTime = DateTime
+      .parse(lastCheckIn.getString("dateTime"));
+
+    assertThat(actualCheckinDateTime, is(secondCheckInDate));
```

Comment:
Please can the variable suffixes be consistent, either `Date` or `DateTime`

Reply:
Ok.